pkg/cfn/manager/tasks_test.go

```diff
@@ -112,14 +112,16 @@ var _ = Describe("StackCollection Tasks", func() {
 				Expect(tasks.Describe()).To(Equal(`1 task: { create cluster control plane "test-cluster" }`))
 			}
 			{
-				tasks := stackManager.NewTasksToCreateClusterWithNodeGroups(makeNodeGroups("bar", "foo"), makeManagedNodeGroups("m1", "m2"))
```

Comment:
Why change this? This should be a new test, right? The original `makeManagedNodeGroups` task didn't disappear, did it?

Reply:
This is true, re-added in 34516fb53077db5587bbcaff86a6ed5566ea1676